weed/command/filer.go

```diff
@@ -91,6 +92,7 @@ func init() {
 	f.showUIDirectoryDelete = cmdFiler.Flag.Bool("ui.deleteDir", true, "enable filer UI show delete directory button")
 	f.downloadMaxMBps = cmdFiler.Flag.Int("downloadMaxMBps", 0, "download max speed for each download request, in MB per second")
 	f.diskType = cmdFiler.Flag.String("disk", "", "[hdd|ssd|<tag>] hard drive or solid state drive or any tag")
+	f.allowedOrigins = cmdFiler.Flag.String("allowedOrigins", "*", "comma separated list of allowed origins")
```

Comment:
how about defaulting `allowedOrigins` to an empty string?

Reply:
The filer and S3 gateway will need a value of an address or the wildcard on startup in order to work properly. With a blank value the requests from the browser will fail unless they are from the same origin I believe. Requests from a non web could would most likely continue to work.